src/main/java/com/dnastack/ddap/explore/wes/client/AccessErrorException.java

```diff
@@ -0,0 +1,18 @@
+package com.dnastack.ddap.explore.wes.client;
+
+import lombok.Getter;
+
+public class AccessErrorException extends RuntimeException {
```

Comment:
There is class `org.springframework.web.server.ResponseStatusException`, which suits our needs here.